tombstone_gc.hh

```diff
@@ -28,23 +30,36 @@ class database;
 
 }
 
-class repair_history_map;
-using per_table_history_maps = std::unordered_map<table_id, seastar::lw_shared_ptr<repair_history_map>>;
+using reconcile_history_map = boost::icl::interval_map<dht::token, gc_clock::time_point, boost::icl::partial_absorber, std::less, boost::icl::inplace_max>;
```

Comment:
Please add a comment what we store in the map and what's it used for

Reply:
added